config/default/field.field.node.hs_basic_page.field_hs_page_hero.yml

```diff
@@ -28,77 +28,71 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
-      hs_banner: hs_banner
-      hs_hero_image: hs_hero_image
       hs_gradient_hero_slider: hs_gradient_hero_slider
       hs_carousel: hs_carousel
+      hs_banner: hs_banner
+      hs_hero_image: hs_hero_image
       hs_sptlght_slder: hs_sptlght_slder
     negate: 0
     target_bundles_drag_drop:
       hs_accordion:
-        weight: -40
+        weight: -38
         enabled: false
       hs_banner:
-        weight: -45
+        weight: -41
         enabled: true
       hs_callout_box:
-        weight: -32
+        weight: -30
         enabled: false
       hs_carousel:
         weight: -42
         enabled: true
       hs_clr_bnd:
-        weight: -26
+        weight: -24
         enabled: false
       hs_collection:
-        weight: -31
+        weight: -29
         enabled: false
       hs_gradient_hero:
-        weight: -35
+        weight: -33
         enabled: false
       hs_gradient_hero_slider:
         weight: -43
         enabled: true
       hs_hero_image:
-        weight: -44
+        weight: -40
         enabled: true
       hs_postcard:
-        weight: -39
+        weight: -37
         enabled: false
       hs_priv_collection:
-        weight: -25
+        weight: -23
         enabled: false
       hs_priv_text_area:
-        weight: -38
+        weight: -36
         enabled: false
       hs_row:
-        weight: -37
-        enabled: false
-      hs_spotlight:
-        weight: -24
+        weight: -35
         enabled: false
       hs_sptlght_slder:
-        weight: -41
+        weight: 0
         enabled: true
       hs_testimonial:
-        weight: -30
+        weight: -28
         enabled: false
       hs_text_area:
-        weight: -36
+        weight: -34
         enabled: false
       hs_timeline:
-        weight: -29
+        weight: -27
         enabled: false
       hs_timeline_item:
-        weight: -28
+        weight: -26
         enabled: false
       hs_view:
-        weight: -34
-        enabled: false
-      hs_webform:
```

Comment:
In overridden config, try to include only the lines necessary for the the `hs_webform` paragraph removal. All other changes might be different from site to site and it's better to leave them out of the PR.

Reply:
Removed other changes.